src/rgw/rgw_sal_motr.cc

```diff
@@ -243,7 +243,7 @@ int MotrMetaCache::put(const DoutPrefixProvider *dpp,
                        const string& name,
                        const bufferlist& data)
 {
-  ldpp_dout(dpp, 0) <<__func__ << ": Put into cache: name=" << name << dendl;
+  ldpp_dout(dpp, 20) <<__func__ << ": Put into cache: name=" << name << dendl;
```

Comment:
lets use `LOG_DEBUG` instead of int value.
```suggestion
  ldpp_dout(dpp, LOG_DEBUG) <<__func__ << ": Put into cache: name=" << name << dendl;
```

Reply:
done

Reply:
yes, address this elsewhere too